bin/load-runner.js

```diff
@@ -453,9 +459,14 @@ var before = function(beforeScript, cb) {
 
 var startTime;
 
+var finish = function(summary) {
```

Comment:
Could this be `function finish(summary) {`? That way, it wouldn't be an anonymous function (and I think it could even stay down where it originally was before you moved it up here, and eslint would be fine).

I'm not sure if that makes it a constant function (we never want to have to worry about someone else making a different thing called `finish`, do we?). If it doesn't, maybe `const finish = function finish(summary) {`, might be a little better. If that's the one that you go with, you may need to leave it here for eslint, I'm not sure.